dace/codegen/targets/intel_fpga.py

```diff
@@ -1067,15 +1068,17 @@ def generate_memlet_definition(self, sdfg, dfg, state_id, src_node,
 
         result = ""
 
+        is_global = False
```

Comment:
`is_global` means that this is `FPGA_Global`, or that it is allocated in the top-level scope?

Reply:
The latter